db/keyval/redis/bytes_broker_impl.go

```diff
@@ -0,0 +1,547 @@
+// Copyright (c) 2017 Cisco and/or its affiliates.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package redis
+
+import (
+	"time"
+
+	"github.com/ligato/cn-infra/db/keyval"
+	"github.com/ligato/cn-infra/logging"
+	"github.com/ligato/cn-infra/logging/logrus"
+
+	"errors"
+	"reflect"
+	"strings"
+
+	"bytes"
+
+	"fmt"
+	"github.com/garyburd/redigo/redis"
+)
+
+// BytesConnectionRedis allows to store, read and watch values from Redis.
+type BytesConnectionRedis struct {
+	pool *redis.Pool
```

Comment:
Could you please use here interface to be able to replace normal pool with cluster enabled pool?